cime/config/e3sm/machines/config_batch.xml

```diff
@@ -398,7 +398,8 @@
 
    <batch_system MACH="compy" type="slurm">
     <queues>
-      <queue walltimemax="00:59:00" default="true">slurm</queue>
+      <queue walltimemax="02:00:00" nodemax="40" default="true">short</queue>
```

Comment:
One more comment, I believe you want to add
```
strict="true"
```
to the short queue definition.

Reply:
Ok, I will do that. Just for my understanding, what does this "strict" keyword do? From its name, it sounds like it will enforce everything strictly. I am not sure how the "non-strict" version decides what to allow and what not to allow. May be I am completely misinterpreting this variable.